30.2/Oznaczamy bestsellery

```diff
@@ -0,0 +1,47 @@
+SHOW COLUMNS FROM BOOKS LIKE 'BESTSELLER';
+
+ALTER TABLE BOOOKS ADD BESTSELLER BOOLEAN DEFAULT false;
+
+DROP PROCEDURE IF EXISTS UpdateBestsellers;
+DELIMITER $$
+
+CREATE PROCEDURE UpdateBestsellers()
+BEGIN
+DECLARE BOOKSRENTED, DAYS, BKS_ID INT;
+DECLARE FINISHED INT DEFAULT 0;
+DECLARE ALL_BOOKS CURSOR FOR SELECT BOOK_ID FROM BOOKS;
+DECLARE CONTINUE HANDLER FOR NOT FOUND SET FINISHED = 1;
+	OPEN ALL_BOOKS;
+    WHILE (FINISHED = 0) DO
```

Comment:
co to za warunek? wygląda jak nieskończona pętla

Reply:
Tak jak w submodule, aby zabezpieczyć się przed brakiem kolejnego argumentu, gdy ustawimy kursor przed ostatnim argumentem następnie fetch przesynie go na koniec, wtedy zmienimy wartość finished na 1 i while() nie spełni warunku